CHANGES.txt

```diff
@@ -273,6 +273,7 @@
  * Add ELAPSED command to cqlsh (CASSANDRA-18861)
  * Add the ability to disable bulk loading of SSTables (CASSANDRA-18781)
  * Clean up obsolete functions and simplify cql_version handling in cqlsh (CASSANDRA-18787)
+ * Add configuration for sorted imports in source files (CASSANDRA-17925)
```

Comment:
wrong position. CHANGES entries are prepended, the order reverse-chronological.

Reply:
Fixed.